<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-K2BIR38" ignored="0" name="Test" tests="2" failures="1" timestamp="24 Feb 2018 21:43:03 GMT" time="26.013" errors="0">
  <testcase name="negativeTest" time="13.909" classname="com.agile.testng.SmokeTest">
    <failure type="java.lang.AssertionError" message="expected [Validation errors occurred. Please confirm the fields and submit it again.] but found [Please fill in the required field.]">
      <![CDATA[java.lang.AssertionError: expected [Validation errors occurred. Please confirm the fields and submit it again.] but found [Please fill in the required field.]
	at com.agile.pages.ContactPage.verifyErrorAlert(ContactPage.java:58)
	at com.agile.testng.Base.negativeEndToEnd(Base.java:57)
	at com.agile.testng.SmokeTest.negativeTest(SmokeTest.java:21)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- negativeTest -->
  <testcase name="positiveTest" time="12.058" classname="com.agile.testng.SmokeTest"/>
</testsuite> <!-- Test -->
